service/matching/tasklist/testing.go

```diff
@@ -0,0 +1,375 @@
+// The MIT License (MIT)
```

Comment:
this file path is not in the exclusions. can you move it to testing/ or testdata/ folder?